ukraine and refugee expanders on non-uk pages (#4449)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

navigation_description: Learn more about teacher training fees in England and financial help for physics and languages trainees.
image: "static/images/content/hero-images/0034.jpg"
backlink: "../../"
expander:
  international-content:

>     header: Refugees and asylum seekers

Author: ```suggestion
    header: Refugees and asylum seekers:
```

>     header: Refugees and asylum seekers:

Author: ```suggestion
    header: Refugees and asylum seekers
```

    title: funding support
    text:

>       "If you <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees/non-uk-teachers/non-uk-qualifications\">have refugee status you'll ususally be eligible for financial support to help you train</a>.

Author: ```suggestion
      "If you <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">have refugee status you'll usually be eligible for financial support to help you train</a>.
```

navigation_description: Learn more about teacher training fees in England and financial help for physics and languages trainees.
image: "static/images/content/hero-images/0034.jpg"
backlink: "../../"
expander:
  international-content:

>     title: funding support

Reviewer: missing a colon between header and title

>     text: If you <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">have refugee status you'll usually be eligible for financial support to help you train</a>.

Reviewer: link doesn't work (strange slash at start?)

>       <p>If you're in <a href=\"https://www.gov.uk/claim-asylum\"> the process of seeking asylum</a>, check your immigration bail conditions to see if you have permission to study on a teacher training course in England. Even if you have permission, you're unlikely to be eligible for financial support.</p>

Reviewer: link doesn't work (strange slash at start?)

>       <p>If you're in <a href=\"https://www.gov.uk/claim-asylum\"> the process of seeking asylum</a>, check your immigration bail conditions to see if you have permission to study on a teacher training course in England. Even if you have permission, you're unlikely to be eligible for financial support.</p>

Reviewer: ```suggestion
      <p>If you're in <a href=\"https://www.gov.uk/claim-asylum\"> the process of seeking asylum</a>, check your immigration bail conditions to see if you have permission to study on a teacher training course in England. Even if you have permission, you're unlikely to be eligible for financial support.</p>
```

      <p>If you're granted asylum in the UK, you may be eligible for financial support to train to teach.</p>
    header: Refugees and asylum seekers

>     expanded: true

Reviewer: not sure why you have this - this means the expander is open by default?

navigation_description: Learn more about teacher training fees in England and financial help for physics and languages trainees.
image: "static/images/content/hero-images/0034.jpg"
backlink: "../../"
expander:
  refugees-asylum-seekers:

>     header: Refugees and asylum seekers

Author: ```suggestion
    header: "Refugees and asylum seekers:"
```

    title: funding support
    text: |- 

>       <p>If you <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">have refugee status you'll usually be eligible for financial support to help you train</a>.

Reviewer 2: This link sends users to the page they're already on?

>       <p>If you <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">have refugee status you'll usually be eligible for financial support to help you train</a>.

Author: ```suggestion
      <p>If you <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">have refugee status you'll usually be eligible for financial support to help you train</a>.
```

* you have some type of settled status in the UK – for example, through the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain)

> * you're [from the Ukraine or Afghanistan and are on certain visas](/non-uk-teachers/visas-for-non-uk-trainees)

Author: ```suggestion
* you're [from Ukraine or Afghanistan and are on certain visas](/non-uk-teachers/visas-for-non-uk-trainees)
```

* you have some type of settled status in the UK – for example, through the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain)

> * you're [from Ukraine or Afghanistan and are on certain visas](/non-uk-teachers/visas-for-non-uk-trainees)

Reviewer 2: Should this be 'have certain visas'?

* tuition fee loans
* maintenance loans to help with living costs

> * extra help if you have a disability, children or an adult dependent

Reviewer: I think `dependant` is correct - ie no change is needed?

Author: Yep it's dependant so no change needed

> * extra help if you have a disability, children or an adult dependent

Author: ```suggestion
* extra help if you have a disability, children or an adult dependant
```
